specification/cognitiveservices/AnomalyDetector/client.tsp

```diff
@@ -3,4 +3,4 @@ import "./main.tsp";
 
 using Azure.ClientGenerator.Core;
 
-@@convenientAPI(AnomalyDetector.Univariate.DetectUnivariateEntireSeries, false)
+@@convenientAPI(AnomalyDetector.Univariate.detectUnivariateEntireSeries, false)
```

Comment:
Why change to lower camelCase?

Reply:
because, otherwise it runs afoul of linting rules, and this really should be camelCase